src/app/modals/property-management-modal/property-management-modal.component.html

```diff
@@ -0,0 +1,163 @@
+<div class="modal d-block">
+  <div class="modal-dialog modal-dialog-scrollable">
+    <div class="modal-content position-fixed top-0 end-0" style="width: 30rem; min-height: 100vh;"
```

Comment:
This style can't use the "generic" css class that we have created? (id="modal-content")
This way we can be sure that all modals have the same size.